lib/taste_tester/tunnel.rb

```diff
@@ -44,15 +44,13 @@ def run
     end
 
     def cmd
-      if TasteTester::Config.user != 'root'
-        pid = '$$'
-      else
-        pid = '\\$\\$'
```

Comment:
@natewalck  ... not on Linux when you're going directly, right? That's what I need you to test.